config/default.json

```diff
@@ -46,6 +46,7 @@
         "RestrictPostDelete": "all",
         "AllowEditPost": "always",
         "PostEditTimeLimit": 300,
+        "EnableAuthenticationTransfer": true ,
```

Comment:
Can you help change config setting name to `ExperimentalEnableAuthenticationTransfer`
 - we may replace it with a different setting later. For instance, guest accounts may use a different authentication method from internal members. So let's add `Experimental` for now so admins know it might get replaced/removed

Also, looks like there's an extra space after `true`.